# Publish all joint states
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

# Joints controllers, each joint with individual command
base_effort_controller:
  type: effort_controllers/JointPositionController
  joint: base
  pid:
      d: 20 # 10
      i: 0
      p: 50

shoulder_effort_controller:
  type: effort_controllers/JointPositionController
  joint: shoulder
  pid:
      d: 100 # 10
      i: 0  # 0
      p: 1000   # 1000

elbow_effort_controller:
  type: effort_controllers/JointPositionController
  joint: elbow
  pid:
      d: 50 # 10
      i: 1 # 1
      p: 1000 #500 #30

pitch_effort_controller:
  type: effort_controllers/JointPositionController
  joint: pitch
  pid:
      d: 10
      i: 0
      p: 100

roll_effort_controller:
  type: effort_controllers/JointPositionController
  joint: roll
  pid:
      d: 10
      i: 1
      p: 50

gripper_finger_left_effort_controller:
  type: effort_controllers/JointPositionController
  joint: gripper_finger_left_joint
  pid:
      d: 10
      i: 1
      p: 100

gripper_finger_right_effort_controller:
  type: effort_controllers/JointPositionController
  joint: gripper_finger_right_joint
  pid:
      d: 10
      i: 1
      p: 100



# Joint trajectory controller, all joints controller
trajectory_controller:
  type: effort_controllers/JointTrajectoryController
  joints:
    - base
    - shoulder
    - elbow
    - pitch
    - roll
    - gripper_finger_left_joint
    - gripper_finger_right_joint


